Cargo.toml

```diff
@@ -53,4 +54,4 @@ translator_common = { path = "examples/translator/common" }
 expect = { path = "third_party/expect" }
 grpc-compiler = { path = "third_party/grpc-rust/grpc-compiler" }
 protoc-rust-grpc = { path = "third_party/grpc-rust/protoc-rust-grpc" }
-rust = { path = "third_party/rust/src/libstd" }
+rust_asylo = { path = "third_party/rust/src/libstd" }
```

Comment:
I'm a bit confused as to what should go in `eal_asylo` and what in `rust_asylo` now, could you clarify? I guess that's part of #656, so perhaps it makes sense to reply there?

Reply:
- rust_asylo is in third_party as its derived from rust_lang/rust, and intends to be as close to standard as possible
- oak_platform_rust is an implementation of oak_platform for asylo that uses rust_asylo

I agree there is overlap here. oak_platform_rust could be merged into either rust_asylo or as oak_platform. Currently `oak_platform_rust` is mostly just what was previously in `oak/asylo_rust`. This is worth revisiting when the implementation of "eal like things" becomes clearer, I've added `TODO(#662)`.

#656 is this PR did you mean to link something else?

Reply:
Uhmm I don't remember which issue I meant to point to, anyways I think #662 covers it now, thanks!